packages/data-stores/src/plans/hooks/use-pricing-meta-for-grid-plans.ts

```diff
@@ -56,6 +56,12 @@ interface Props {
 	 * If true, the pricing includes discounts from upgrade credits.
 	 */
 	withProratedDiscounts?: boolean;
+
+	/**
+	 * TODO: Add more detailed comments because we've decided to make a tempoarary decision around storage-addons
+	 * while the calypso_plans_page_emphasize_longer_plan_savings experiment is running.
+	 */
+	enableTermSavingsPriceDisplay?: boolean;
```

Comment:
If we do keep this prop, I think it's best to rename it to something else here. Maybe, `reflectStorageSelection` or `reflectStorageSelectionInPricing`? Only for the pricing hook (`usePricingMetaForGridPlans`), it's fine for the grid components otherwise.

Reply:
I see where you're going, and I like the intention of renaming a lot. The term `reflection` throws me off a bit. I'll think a bit more on it.

Non-urgent question ( more for my learning than anything ). Was there reasoning that helped you recognize that we should update the name?

Was it some sort of heuristic? Something like parameter names, at this level of abstraction, should be more generalized?

Reply:
Oh hey quick note about my comment "Non-urgent question ( more for my learning than anything ). Was there reasoning that helped you recognize that we should update the name?"

I'm rereading the sentence and my mistake if the tone came off as skeptical -- I actually totally agree with the renaming. I was honestly wondering how you spotted it so I can learn to recognize the same thing in the future 🙂

Reply:
Ah yeah, totally read it like that 😃 

So no worries. Request for change was mainly for billing description thing, in case we miss it